packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -409,7 +409,8 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
 
   private _getBounds(): IRectangle {
     if (!this._bounds) {
-      let currentBounds = this.props.bounds;
+      const bounds = this.props.bounds;
+      let currentBounds = typeof bounds === 'function' ? bounds(this.props.target) : bounds;
```

Comment:
>this.props.target [](start = 64, length = 17)

questionable if this should be this._target instead but I think we could maybe get into a situation where this._target hasn't been set get even when there's a this.props.target

Reply:
After looking at this again, I think this is the right fix. Making any assumption on if this._target has already been set doesn't sounds like a good idea (and that value is also just determined from this.props.target so it would really get us anything anyways)

Reply:
you should also pass in this._targetWindow since the callout does some work to ensure it has the right window associated with the target.